From 3a38220529745a4e270e70be81a8501d07a98e05 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Wed, 15 Sep 2021 16:19:16 +0200
Subject: [PATCH 210/529] rtc: stm32: fix unnecessary parentheses

Fix a few style issues reported by checkpatch.pl:
- Unnecessary parentheses
- Lines should not end with a '('

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Ifdfc169560f767d9d51b33bf6a60d231cae8fc18
---
 drivers/rtc/rtc-stm32.c | 29 ++++++++++++++---------------
 1 file changed, 14 insertions(+), 15 deletions(-)

--- a/drivers/rtc/rtc-stm32.c
+++ b/drivers/rtc/rtc-stm32.c
@@ -160,10 +160,9 @@ static int stm32_rtc_enter_init_mode(str
 		 * slowest rtc_ck frequency may be 32kHz and highest should be
 		 * 1MHz, we poll every 10 us with a timeout of 100ms.
 		 */
-		return readl_relaxed_poll_timeout_atomic(
-					rtc->base + regs->isr,
-					isr, (isr & STM32_RTC_ISR_INITF),
-					10, 100000);
+		return readl_relaxed_poll_timeout_atomic(rtc->base + regs->isr, isr,
+							 (isr & STM32_RTC_ISR_INITF),
+							 10, 100000);
 	}
 
 	return 0;
@@ -448,16 +447,16 @@ static int stm32_rtc_valid_alrm(struct s
 	 *	M-D-Y H:M:S < alarm <= (M+1)-D-Y H:M:S
 	 * with a specific case for December...
 	 */
-	if ((((tm->tm_year > cur_year) &&
-	      (tm->tm_mon == 0x1) && (cur_mon == 0x12)) ||
-	     ((tm->tm_year == cur_year) &&
-	      (tm->tm_mon <= cur_mon + 1))) &&
-	    ((tm->tm_mday > cur_day) ||
-	     ((tm->tm_mday == cur_day) &&
-	     ((tm->tm_hour > cur_hour) ||
-	      ((tm->tm_hour == cur_hour) && (tm->tm_min > cur_min)) ||
-	      ((tm->tm_hour == cur_hour) && (tm->tm_min == cur_min) &&
-	       (tm->tm_sec >= cur_sec))))))
+	if (((tm->tm_year > cur_year &&
+	      tm->tm_mon == 0x1 && cur_mon == 0x12) ||
+	     (tm->tm_year == cur_year &&
+	      tm->tm_mon <= cur_mon + 1)) &&
+	    (tm->tm_mday > cur_day ||
+	     (tm->tm_mday == cur_day &&
+	     (tm->tm_hour > cur_hour ||
+	      (tm->tm_hour == cur_hour && tm->tm_min > cur_min) ||
+	      (tm->tm_hour == cur_hour && tm->tm_min == cur_min &&
+	       tm->tm_sec >= cur_sec)))))
 		return 0;
 
 	return -EINVAL;
@@ -666,7 +665,7 @@ static int stm32_rtc_init(struct platfor
 	 * Can't find a 1Hz, so give priority to RTC power consumption
 	 * by choosing the higher possible value for prediv_a
 	 */
-	if ((pred_s > pred_s_max) || (pred_a > pred_a_max)) {
+	if (pred_s > pred_s_max || pred_a > pred_a_max) {
 		pred_a = pred_a_max;
 		pred_s = (rate / (pred_a + 1)) - 1;
 
